services/movie.py

```diff
@@ -0,0 +1,34 @@
+from db.models import Movie
+
+
+def get_movies(
+    genres_ids: list[int] = None, actors_ids: list[int] = None
```

Comment:
```suggestion
    genres_ids: Optional[list[int]] = None, actors_ids: Optional[list[int]] = None
```
Because None by default. Fix in all places